wa/commands/create.py

```diff
@@ -77,7 +125,91 @@ class CreateCommand(ComplexCommand):
     object-specific arguments.
     '''
     subcmd_classes = [
-        #CreateWorkloadSubcommand,
-        #CreatePackageSubcommand,
+        CreateWorkloadSubcommand,
         CreateAgendaSubcommand,
+        #CreatePackageSubcommand,
     ]
+
+
+def create_workload(name, kind='basic', where='local', check_name=True, **kwargs):
+
+    create_funcs = {
```

Comment:
Rather than listing the workloads in `workload_types` class attribute, and then again here, I would move this dict into global level and the use its keys instead of `workload_types`

Reply:
Thanks, updated.